### DISCUSSION TITLE
[Question][Hw0] Installing Git

### DISCUSSION THREAD

**Discussion by nybunny** [prosyslab-classroom/cs348-information-security#480] [Q&A]
When I connect to KCLOUD VM (using Putty), the default place is
      root@[hostname]:~#
then when I type cd .. ("root@[hostname]:~# cd ..")
      root@c[hostname]:/#
this comes out.
Q1) What is the difference between "root@[hostname]:~#" and "root@c[hostname]:/#"?
      (Is the former a hidden folder?)
Q2) Where should I install git (i.e. type "sudo apt update" and "sudo install git") - and later set up my information?

(and why does a cancel line appear?)

**Comment by nv-quan** [comment]
For the strike-through line, it appears because of this symbol `~`.

**Comment by RiceBiscuits** [comment]
Q1) '~' means home directory. 
When you access your account, directory position is home directory of your account.
You can type 'pwd' command and check your current position.
Then, you can find that Root account's home directory is '/root'.
Thus, the result of 'cd ..' command will be / (root directory of the filesystem).

**Comment by RiceBiscuits** [comment]
Q2) You can type install commands anywhere you want. They're not saved in your working directory. 
After installing Git, you can type 'dpkg -L git' to find where Git package is located.
commands which set git account information are depends on location and options such as '--global', '--local'.
'git config --global user.name "Your Name" ' will affect all git repository saved in your storage regardless of your working directory.

**Comment by sumango3** [comment]
If the problem is solved, please close this issue.
